rfcs/0008-node-discovery/0008-node-discovery.md

```diff
@@ -0,0 +1,92 @@
+---
+Number: 8
+Category: Standards Track
+Status: Draft
+Author: Linfeng Qian, JinYang Jiang
+Organization: Nervos Foundation
+Created: 2018-11-28
+---
+
+# CKB Node Discovery Protocol
+
+CKB node discovery protocol mainly refers to [Satoshi Client Node Discovery][0]. The differences between them are summarized below:
+
+* The node version number is included in the `GetNodes` message.
+* The `Nodes` message is used to periodically broadcast all nodes currently connected.
+* We use `multiaddr` as the format of node addresses (It MUST NOT include `/p2p/` segment otherwise it's considered as *misbehavior* and a low score will be given.)
+
+Every time client startup, if PeerStore's address list is empty, it will try to issue DNS requests to initialize address list, if DNS requests failed it will fallback to the hard-coded address list.
+
+## Discovery Methods
+### DNS Addresses
+When first time startup (bootstrap stage), if discovery service is needed, local node then issues DNS requests to learn about the addresses of other peer nodes. The client includes a list of host names for DNS services that are seeded.
```

Comment:
Can we add a definition for `seeded`, i.e. what kind of nodes can be called `seeded`?

Reply:
Added